common/src/main/res/values/strings.xml

```diff
@@ -1274,4 +1275,10 @@
     <string name="state_update_available">Update available</string>
     <string name="state_up_to_date">Up-to-date</string>
     <string name="sensor_name_health_connect">Health Connect sensors</string>
+    <string name="graph_state_widgets">Graph historic Widget</string>
```

Comment:
Could we perhaps remove widget entirely and make it more like the other widgets? "History graph" sounds nice, but may be confusing because of the frontend card of the same name, or "Sensor graph".

(To me as a non-native speaker, "graph historic widget" doesn't sound right.)